src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -87,50 +87,78 @@ public void visit(NodeTraversal t, Node n, Node parent) {
           }
           break;
         case GETPROP:
-          // Converts a class inner typedef into a top level interface, which then later has its
-          // members converted in TypeAnnotationPass.
+          // Converts a class inner typedef into either (1) a top level interface, which then later
+          // has its members converted in TypeAnnotationPass; or (2) a type alias.
           // Most class inner @typedef meant @record in closure but they were added before @record
           // was supported. Also in TypeScript interfaces are preferred to type alias because of
-          // better error reporting and ability to extend.
+          // better error reporting and ability to extend. However simple types such as string are
+          // still type aliases.
           bestJSDocInfo = NodeUtil.getBestJSDocInfo(n);
           if (bestJSDocInfo != null && bestJSDocInfo.hasTypedefType()) {
```

Comment:
could you invert this condition and early exit with a `break` or so, instead of adding more and more indent?